ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx

```diff
@@ -184,7 +184,7 @@ const ItemTableContainer = ({
               anchorEl={anchorEl}
             />
             {hideableColumns.length > 1 && (
-              <Tooltip title={t('label.toggle_visible_columns')} placement="bottom">
+              <Tooltip title={t('label.toggle_visible_columns') || ''} placement="bottom">
```

Comment:
Whats the point of this? We should have the translations loaded for this right?

Reply:
No clue why `TS` was yelling about insufficient types from `Tooltip`. It looked like `t()` is undefined on initialization hence the errors but that shouldn't be true to how we load the translations 🤔

Reply:
Strange indeed, maybe some types got updated, either way this is fine I guess, we can revisit this later to see if we can clean it up!